gradle.properties

```diff
@@ -63,3 +63,4 @@ android.useAndroidX=true
 org.gradle.configureondemand=false
 org.gradle.daemon=true
 org.gradle.jvmargs=-Xmx4096M
+org.gradle.java.home=/Users/bryonnabaines/Library/Java/JavaVirtualMachines/corretto-1.8.0_302/Contents/Home
```

Comment:
Should this be in this file? Seems very specific to your device and won't function for anybody but you I would assume :)

Reply:
Nice catch, I moved it to local.properties instead